src/tools/genpolicy/src/registry_containerd.rs

```diff
@@ -27,6 +27,11 @@ use tonic::transport::{Endpoint, Uri};
 use tonic::Request;
 use tower::service_fn;
 
+const TAR_LAYER_TYPE: &str = "application/vnd.oci.image.layer.v1.tar";
+const TAR_DIFF_LAYER_TYPE: &str = "application/vnd.docker.image.rootfs.diff.tar";
```

Comment:
Does this type exist? I only find references to `application/vnd.docker.image.rootfs.diff.tar.gzip`.